assets/javascripts/kitten/components/steppers/vertical-stepper/components/title.js

```diff
@@ -1,11 +1,41 @@
+import React from 'react'
 import styled from 'styled-components'
-import { pxToRem } from '../../../../helpers/utils/typography'
+import PropTypes from 'prop-types'
+import { pxToRem, stepToRem } from '../../../../helpers/utils/typography'
 import TYPOGRAPHY from '../../../../constants/typography-config'
+import classNames from 'classnames'
 
-export const Title = styled.p`
+const StyledTitle = styled.p`
   margin: 0 0 ${pxToRem(2)};
 
   ${TYPOGRAPHY.fontStyles.regular};
-  font-size: ${pxToRem(16)};
+  font-size: ${stepToRem(0)};
   line-height: normal;
+
+  &.k-Stepepers-VerticalStepper__title--orion {
+    font-size: ${stepToRem(-1)};
+    font-weight: 500;
```

Comment:
J'irais plutôt faire :

```suggestion
    {TYPOGRAPHY.fontStyles.regular}
```